apps/publish/enqueue/enqueue_service.py

```diff
@@ -254,9 +254,11 @@ def publish(self, doc, target_media_type=None, content_type=None):
         :raises PublishQueueError.item_not_queued_error:
                 If the nothing is queued.
         """
-        publish_first_version = True
-        if doc.get("rewrite_of") or doc.get("state") == "corrected" or doc.get("translated_from"):
-            publish_first_version = False
+        publish_first_version = False
```

Comment:
what does this mean? new item being published? maybe `new_item` would be better name